api/server/backup.go

```diff
@@ -273,18 +299,31 @@ func (vd *volAPI) cloudBackupSchedCreate(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	d, err := vd.getVolDriver(r)
+	// Get context with auth token
+	ctx, err := vd.annotateContext(r)
 	if err != nil {
-		notFound(w, r)
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	backupSchedResp, err := d.CloudBackupSchedCreate(backupSchedReq)
+	// Get gRPC connection
+	conn, err := vd.getConn()
 	if err != nil {
-		vd.sendError(method, backupSchedReq.SrcVolumeID, w, err.Error(), http.StatusInternalServerError)
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	json.NewEncoder(w).Encode(backupSchedResp)
+
+	volumes := api.NewOpenStorageCloudBackupClient(conn)
+	resp, err := volumes.SchedCreate(ctx, &api.SdkCloudBackupSchedCreateRequest{
+		CloudSchedInfo: sdk.ToSdkCloudBackupdScheduleInfo(api.CloudBackupScheduleInfo{
```

Comment:
did I misspell this api? wow. `Backupd` 😕

Reply:
Oops :)